test/lib/configure-security.test.js

```diff
@@ -1,11 +1,12 @@
+const { expect } = require('chai');
 const configureSecurity = require('../../src/lib/configure-security');
 
-describe('configure-security', () => {
```

Comment:
Our [`testing-mocha` ESLint config](https://github.com/readmeio/standards/blob/main/packages/eslint-config/testing-mocha.js) requires that Mocha `describe` and `it` blocks don't have arrow functions because Mocha discourages it: https://mochajs.org/#arrow-functions